WcaOnRails/config/locales/de.yml

```diff
@@ -2569,48 +2569,71 @@ de:
       payment_successful: >-
         Deine Zahlung war erfolgreich und deine Zahlung sollte unten als bezahlt
         markiert sein.
+      #original_hash: 92dde77
+      payment_processing: 'Deine Zahlung wurde bestätigt, aber die Transaktion steht noch aus.'
+      #original_hash: 62d7351
+      button_text: Zahle jetzt!
       hints:
         #original_hash: bf3a39e
         donation: Zusätzliche Spenden gehen nur an das Organisationsteam
-        #original_hash: 16cf92d
-        card_information: >-
-          Deine Karteninformationen werden nicht an die WCA Website übertragen.
-          Sie werden verschlüsselt an Stripe übertragen, welches die Abwicklung
-          der Zahlung übernimmt und an uns weiterleitet.
+        #original_hash: f7fa263
+        payment_button: >-
+          Deine Zahlungsinformationen gelangen niemals zur WCA Website. Sie
+          werden verschlüsselt an Stripe übertragen, welches den Zahlungsprozess
```

Comment:
Ist Stripe neutrum? Eher "an Stripe übertragen, wo der Zahlungsprozess abgewickelt wird und anschließend eine Weiterleitung an uns erfolgt"

Reply:
"welches" hatten wir in der vorherigen Iteration der Übersetzung auch schon, also Neutrum hatten wir in der Vergangenheit wohl angenommen. Finde aber i.A. den Vorschlag besser und habe ihn dann einfach komplett übernommen.